src/param.rs

```diff
@@ -125,6 +125,18 @@ pub enum Param {
     /// For Groups and Contacts
     ProfileImage = b'i',
 
+    /// For Chats an Contacts
+    ///
+    /// For Chats: If this is a mailing list chat, contains the List-Post address.
+    /// If it's empty, the mailing list is read-only.
+    ///
+    /// For Contacts: If this is the List-Post address of a mailing list, contains
+    /// the List-Id of the mailing list (which is also used as the group id of the chat).
+    ///
+    /// The List-Post address is the email address where the user can write to in order to
+    /// post something to the mailing list.
+    ListPost = b'p',
```

Comment:
It's not that nice that we are using the same param for different things. I did this because we are running out of characters for the different params, and this way we need one character less. But I can also change it.